src/messages/components/LatestMessages.vue

```diff
@@ -19,6 +19,11 @@
         :label="$t('CONVERSATION.REPLIES')"
         :count="unseenThreadsCount > 9 ? '9+' : unseenThreadsCount"
       />
+      <QRouteTab
+        :title="$t('SETTINGS.TITLE')"
+        icon="fas fa-cog"
+        :to="{ name: 'settings', hash: '#notifications' }"
+      />
     </QTabs>
```

Comment:
The point came up (on our last coding session call) that it would be better to have the cog as a small button in the corner, at the moment it's a implemented as a third tab but with no content, so after first clicking it you get sent to the right place, but then when you re-open the menu you can see the "cog" tab selected, with no content:

![emptytab](https://user-images.githubusercontent.com/31616/80095936-12b2f600-8569-11ea-8708-8abaace9fe7b.png)

I had a play around and found a solution I think is nice:

![flex](https://user-images.githubusercontent.com/31616/80096072-4db52980-8569-11ea-956f-191b856b8a63.png)

I'll talk through the changes as a learning process, so maybe you can implement them.

Currently the component (when displayed as a menu) is a vertical stack of two components:
- `<QTabs>`
- `<LatestConversations>` or `<LatestThreads>` depending on which tab is selected

(the `<template>` component does not result in anything in the document)

But to implement it not as a tab, we need a more involved layout like:

```
<QTabs> | <the cog>
<LatestConversations or LatestThreads>
```
To get the `<QTabs>` and the cog to appear horizontally side-by-side, we can wrap them in a div and use _flex_ layout (see [quasar docs](https://quasar.dev/layout/grid/introduction-to-flexbox) on flexbox).

Quasar lets you use flex layout using the class `flex`, so we get...

```html
<div class="flex">
  <QTabs>... stuff ...</QTabs>
  <QBtn
    flat
    round
    icon="fas fa-fw fa-cog"
    :title="$t('SETTINGS.TITLE')"
    :to="{ name: 'settings', hash: '#notifications' }"
  />
</div>
<... the rest>
```

Initially this results in the tabs and the cog pushed all the way to the left, but we can use more flex stuff to say that the tabs should fill all the space (this is called "grow" in flexbox terminology) using the quasar class `col-grow` on the thing that should grow, so the `<QTabs>` in our case, i.e.:

```html
<QTabs
  ... all the existing stuff
  class="k-message-tabs col-grow"
>
```

That's it!

Reply:
Nice! Done now, thanks for fixing this!